bin/test_integration

```diff
@@ -43,12 +44,13 @@ docker-compose up -d test
 rm -rf $CURRENT_DIR/output/*
 
 if [[ "$DEBUG" == "true" ]]; then
-  docker-compose exec test bash
+  docker-compose exec --privileged test bash
```

Comment:
because add here we use privileged. Hint: Otherwise we get operation not permitted